views/js/runner/plugins/tools/magnifier/magnifierPanel.js

```diff
@@ -96,9 +105,12 @@ define([
      * @param {Number} [config.levelStep] - The level increment applied when using the controls + and -
      * @param {Number} [config.baseSize] - The base size used to assign the width and the height according to the zoom level
      * @param {Number} [config.maxRatio] - The ratio for the maximum size regarding the size of the window
+     * @param {Object} dynamicComponentConfig - to overrides the default
+     * @param {jQuery|HTMLElement|String} [dynamicComponentConfig.renderTo] - An optional container in which renders the component
+     * @param {jQuery|HTMLElement|String} [dynamicComponentConfig.draggableContainer] - the DOMElement the draggable component will be constraint in
      * @returns {magnifierPanel} the component (initialized)
      */
-    function magnifierPanelFactory(config) {
+    function magnifierPanelFactory(config, dynamicComponentConfig) {
```

Comment:
Why not having a single config object as input, and then manage internally the meaning and the split?